### ISSUE TITLE
Allow '_' in device names?

### ISSUE THREAD

**Issue by tamberg** [dyne/dowse#55]
Would it be possible to allow '_'in device names, e.g. tamberg_mac?

**Comment by tamberg** [comment]
Or '-' to use only characters [allowed](https://stackoverflow.com/questions/3523028/valid-characters-of-a-hostname#3523068) in DNS hostnames.

**Comment by parazyd** [comment]
You are right in assuming we only allow valid DNS characters, but for symbols, I am not sure if the dnscrypt-proxy plugin we have allows it. @jaromil can you give any insight on this?
